pre_commit/languages/ruby.py

```diff
@@ -28,6 +28,7 @@ def get_env_patch(
 ) -> PatchesT:  # pragma: win32 no cover
     patches: PatchesT = (
         ('GEM_HOME', os.path.join(venv, 'gems')),
+        ('GEM_PATH', os.path.join(venv, 'gems')),
```

Comment:
was this because `GEM_PATH` was set elsewhere?  do you have a reproduction for this (sorry for being paranoid, don't want to blindly commit this without having a reproduction showing that it's a problem in the first place)

Reply:
I use rvm and when executing pre-commit for a ruby hook with the environment variable `GEM_PATH=~/.rvm/gems/ruby-2.6.5@global`, having `gem-wrappers (1.4.0)` installed in that rvm gemset, I get the warning about the gems being not pristine, I'll try to set up a minimal example with a Dockerfile

Reply:
awesome -- I like your reproduction :)

does `('GEM_PATH', UNSET)` do the same?

Reply:
it does -- just tried it out and pushed to your branch!